### ISSUE TITLE
Repository naming

### ISSUE THREAD

**Issue by ksoda** [shiraikenta/kusoapp#1]
It seems that this App is a quiz game, while I think that its repository name has no relation. An appropriate abstraction enables use of the system without reading codebases.

Have you thought about renaming "kuso" to "Vocabulary Quiz?"

**Comment by shiraikenta** [comment]
You are right. To be honest, I was going to delete this repository after finishing creating this application but I forgot about it. 
Thank you for reminding me of this app anyway.
